### ISSUE TITLE
PE-Injector Control Installation error on line 16

### ISSUE THREAD

**Issue by jadedevin13** [JonDoNym/peinjector#5]
./peinjector-control_install.sh 
Reading package lists... Done
Building dependency tree  
Reading state information... Done
python3 is already the newest version.
python3 set to manually installed.
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
./peinjector-control_install.sh: 16: ./peinjector-control_install.sh: 0: not found
Start peinjector-control server

Produced the following results.
![screenshot from 2015-09-22 05 10 18](https://cloud.githubusercontent.com/assets/6671182/10005171/58212540-60e8-11e5-8a78-76e99bca1a43.png)

**Comment by JonDoNym** [comment]
Try to use https, not http, there is no automatic redirection atm.
We'll investigate the error in the install script.

Can you confirm that the server is running?

**Comment by jadedevin13** [comment]
Thanks the server is running.
But when I installed the intercepter.
It says,
  Failed building wheel for cryptography
  Failed building wheel for Pillow
  Failed building wheel for lxml

Refer here for the complete details
(expires in one month)
http://pastebin.com/A5QChRw9

**Comment by SchnorcherSepp** [comment]
Which os do you use?
and which version?

**Comment by jadedevin13** [comment]
Kali Sana

**Comment by JonDoNym** [comment]
We've stumbled across this problem a few times. It's the mitmproxy installation. Something goes wrong while pip tries to resolve the dependencies.

Try to install mitmproxy first, then run the setup script again:
https://mitmproxy.org/doc/install.html

```
sudo apt-get install python-pip python-dev libffi-dev libssl-dev libxml2-dev libxslt1-dev
sudo pip install mitmproxy
```

If errors occur, you'll have to resolve them manually (we've done this many, many times ...).

In most cases it's an incorrect package version, so install the required version (http://stackoverflow.com/a/5226504)  or upgrade the package (pip install -u PACKAGE).

**Comment by jadedevin13** [comment]
Solved by upgrading to python 3.4. Thanks.

**Comment by nguyenhung162004** [comment]
Regarding to installation script: ./peinjector-control_install.sh error: "./peinjector-control_install.sh: 16: ./peinjector-control_install.sh: 0: not found"

Line 16 of "peinjector-control_install.sh" should change from:
if $(python3 -c 'import sys; print(1 if sys.hexversion<=0x03040000 else 0)') ; then

To:
if [ $(python3 -c 'import sys; print(1 if sys.hexversion<=0x03040000 else 0)') -eq 1 ] ; then

The same for script: "pe-injector-interceptor/install/peinjector-interceptor_dependencies.sh" line 16
